### DISCUSSION TITLE
How do you use RipGrep

### DISCUSSION THREAD

**Discussion by jrock2004** [nicknisi/dotfiles#202] [General]
![image](https://user-images.githubusercontent.com/655716/152667397-b3880b14-308d-4e69-a0f8-2b306c9eb309.png)

I noticed this commit for storybook and wonder how ripgrep helps you search those files

**Comment by nicknisi** [comment]
Good question! I use the custom flags like this to often _exclude_ results from my search using the `-T` flag. I often use the the [`:Find`](https://github.com/nicknisi/dotfiles/blob/0418260afd157b9ba92fa7d990c51ed465043d5f/config/nvim/lua/plugins/fzf.lua#L35) command to do this search from within Vim like this:

```
:Find! -t ts -T spec -T stories someFunctionName
```

This way, I get only `.ts`/`.tsx` files that are not `.stories.tsx?` or `.spec.tsx?`.

**Comment by jrock2004** [comment]
Ah that makes sense
